apps/src/codebridge/utils/isDuplicateFileName.ts

```diff
@@ -47,7 +47,8 @@ export const isDuplicateFileName = ({
     if (existingFile) {
       if (
         existingFile.type === ProjectFileType.SUPPORT ||
-        existingFile.type === ProjectFileType.VALIDATION
+        existingFile.type === ProjectFileType.VALIDATION ||
+        existingFile.type === ProjectFileType.SYSTEM_SUPPORT
```

Comment:
Could add unit test(s) for system support file in `isDuplicateFileNameTest` (in a follow-up?).

Reply:
Added!